pkgs/tools/networking/htpdate/default.nix

```diff
@@ -1,26 +1,26 @@
-{ lib, stdenv, fetchurl }:
+{ stdenv, lib, fetchFromGitHub }:
 
 stdenv.mkDerivation rec {
-  version = "1.2.2";
+  version = "1.3.1";
   pname = "htpdate";
 
-  src = fetchurl {
-    url = "http://www.vervest.org/htp/archive/c/${pname}-${version}.tar.xz";
-    sha256 = "0mgr350qwgzrdrwkb9kaj6z7l6hn6a2pwh7sacqvnal5fyc9a7sz";
+  src = fetchFromGitHub {
+    owner = "twekkel";
+    repo = pname;
+    rev = "v1.3.1";
```

Comment:
```suggestion
    rev = "v${version}";
```

Reply:
Alright. So this is the first time I contribute to a public community project, how should I proceed ? 
Fix, rebase and force push so that there is only one commit in the final log or fix, commit and do nothing so that the history is going to make sense ?

Reply:
> So this is the first time I contribute to a public community project

Welcome!

> how should I proceed ?

There should always be exactly 2 commits in this PR `maintainers: add julienmalka` and `htpdate: 1.2.2 -> 1.3.1`, as the maintainer addition is required to be put in separate commit (but not separate PR) according to the manual so we cannot squash merge this.

And sorry there are too many PRs here and we sometimes cannot give very detailed reviews in one time, just added some more nit suggestions. I will test this later.

Reply:
Okay I'll accept your suggestions and then rebase, I hope that's the adequate way to do